core/src/main/scala/org/bitcoins/core/wallet/fee/FeeUnit.scala

```diff
@@ -9,8 +9,14 @@ import org.bitcoins.core.protocol.transaction.Transaction
   */
 sealed abstract class FeeUnit {
   def currencyUnit: CurrencyUnit
+  def *(cu: CurrencyUnit): CurrencyUnit = this * cu.satoshis.toLong
+  def *(int: Int): CurrencyUnit = this * int.toLong
+  def *(long: Long): CurrencyUnit = currencyUnit * long
   def *(tx: Transaction): CurrencyUnit = calc(tx)
-  def calc(tx: Transaction): CurrencyUnit
+  def txSizeForCalc(tx: Transaction): Long
```

Comment:
What does this do? Can't we just use tx.weight?

Reply:
No, that's only for per-vbyte fee units. For a per-byte we need to use byteSize